README.md

```diff
@@ -2546,8 +2530,6 @@ Bookmarks
 
 [9.520/6.860, Fall 2017](http://www.mit.edu/~9.520/fall17/)
 
-[Stanford University: Tensorflow for Deep Learning Research](http://web.stanford.edu/class/cs20si/index.html)
```

Comment:
This link is still active, so I'm not sure why it should be removed?

It could have been down temporarily at the time, but it is up now.